packages/appsync-emulator-serverless/dynamodbUtil.js

```diff
@@ -5,6 +5,7 @@ const { DynamoDB } = require('aws-sdk');
 async function deriveDynamoDBClient(
   { DynamoDB: config },
   emulatorPath = process.cwd(),
+  port,
```

Comment:
This was only specifying two parameters when it was actually receiving three arguments from `bin/server`. That third argument is the dynamodb port specified via CLI.